website/index.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <!-- Global Metadata -->
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no" />
+  <link rel="shortcut icon" href="favicon.svg" />
+  <title>DEQ Spills Map</title>
+  <meta name="title" content="DEQ Spills Map" />
+  <meta name="description" content="A web mapping application for showing spill report locations." />
+  <!-- Preloads -->
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Source+Sans+3&display=swap" rel="stylesheet" />
```

Comment:
are we using a custom font for this app or is this for the utah header? You'll have to remind me if there is a utah ds website and an embedded component for this app.

Reply:
I assume that this is for the header. I did bring the fonts over from atlas. I expect to show the header when the map is loaded outside of the salesforce iframe.